setup.py

```diff
@@ -26,8 +26,20 @@
             "plugin_manifest.json",
         ],
     },
-    install_requires=["OpenTimelineIO >= 0.12.0"],
-    extras_require={"dev": ["black", "pytest", "twine"]},
+    install_requires=[
+        "OpenTimelineIO >= 0.12.0",
+        "openassetio >= 1.0.0a7",
+        "openassetio-mediacreation >= 1.0.0a2",
```

Comment:
[question] Did we say we were going to pin to concrete versions during alpha? To avoid picking up breaking changes until we have SerVer.

Reply:
We did.

I'm not sure our logic applies to non `openassetio` repos (ie, OpenTimelineIO,) so i'm tenetively leaving that as a `>=` dependency.

Reply:
[ddc907d](https://github.com/OpenAssetIO/otio-openassetio/pull/17/commits/ddc907ded94fcd21e4df6d4978e9a8a66d100146)